packages/relay/tests/lib/openrpc.spec.ts

```diff
@@ -196,6 +196,7 @@ describe("Open RPC Specification", function () {
     });
 
     it('should execute "eth_estimateGas"', async function () {
+        mock.onGet(`accounts/undefined`).reply(404);
```

Comment:
not sure why this is necessary?

Reply:
I also had to read into this but I believe it's since there's no value for `transaction.to` object, so when it's put into the UT here it results in the `undefined` value literally getting stringified.

Could be better structured with a default address but fortunately we do have coverage in acceptances tests where address is a valid hex address